integration_tests/src/main/python/string_test.py

```diff
@@ -401,6 +402,14 @@ def test_length():
                 'CHAR_LENGTH(a)',
                 'CHARACTER_LENGTH(a)'))
 
+@pytest.mark.skipif(is_before_spark_330(),
+                    reason="octet_length and bit_length only available in 330+")
```

Comment:
```suggestion
```
Also should remove the (now unnecessary) import change.

Reply:
Yes, it was removed now.